environment/README.MD

```diff
@@ -1 +1,128 @@
-# environment
\ No newline at end of file
+# Environment
+It is a file about how to prepare environment.
+
+## Table of contents
+* [Download python](#download-python)
+* [Download pycharm](#download-pycharm)
+* [Download git](#download-git)
+* [Install Notepad++](#install-Notepad++)
+* [Create repository on github](#create-repository-on-github)
+* [Clone repository](#clone-repository-using-git-bash)
+* [Create a new branch](#create-a-new-branch)
+* [Create prepare_environment file](#create-prepare_environment-file)
+
+
+
+## Download python
+
+```
+1. Go to https://www.python.org/downloads/
+2. Download python 3.10.4
+3. Install it.
+4. Click windows button+r on your keyboard.
+5. Please provide "cmd" command
+6. Please use command "python3 --version" in cmd
+7. Check version of installed python.
+```
+
+## Download pycharm
+
+```
+1. Go to PyCharm
```

Comment:
where link to Pycharm download, add as a link

Reply:
Done